### DISCUSSION TITLE
How to cite your code?

### DISCUSSION THREAD

**Discussion by JulKoch** [neuralhydrology/neuralhydrology#46] [General]
I was wondering how to properly cite the neuralhydrology code? 

Again, thanks for making it so accessible. Really easy to use.

**Comment by kratzert** [comment]
Hi @JulKoch 

thanks for the question. We actually planned for a long time to submit a paper to the [Journal of Open Source Software](https://github.com/openjournals/joss). Given your request, we decided to work on this right now, which should hopefully not take too long (at least to get a pre-print). Until when do you need to have a reference? We could add an "in prep." bibtex reference, if that would be of help for you.

**Reply by JulKoch** [reply]
Thanks @kratzert for your reply. I am writing on a manuscript right now and I will make sure to update/add the right reference once available. 

If you can come up with a "in preparation" reference now then I would be happy to use that one for the review and update later.

**Comment by gauchm** [comment]
Just to wrap up this thread in case anyone stumbles across it in the future:
With v1.2.3, there is now a [NeuralHydrology paper](https://joss.theoj.org/papers/10.21105/joss.04050#) in the Journal of Open Source Software, and the updated [NeuralHydrology README](https://github.com/neuralhydrology/neuralhydrology#cite-neuralhydrology) provides a bibtex entry.

(sorry it took us so long to get this done)
